types/knex-cleaner/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for knex-cleaner 1.3
+// Project: https://github.com/steven-ferguson/knex-cleaner
+// Definitions by: Karol Goraus <https://github.com/Szarlus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+import * as Knex from 'knex';
+
+export interface KnexCleanerOptions {
+    mode?: 'truncate' | 'delete'; // Default is 'truncate
+    restartIdentity?: boolean; // Used to tell PostgresSQL to reset the ID counter, default is true
+    ignoreTables?: string[]; // Default is empty array
+}
+
+export function clean(knex: Knex, options?: KnexCleanerOptions): Promise<any>;
```

Comment:
Do I assume correctly, that nothing is resolved in that promise? If so, then it can be changed to `Promise<void>` to indicate that.

Reply:
done